src/main/java/magicbees/tileentity/TileEntityApiamancersDrainerGT.java

```diff
@@ -0,0 +1,102 @@
+package magicbees.tileentity;
+
+import java.util.stream.Collectors;
+
+import net.minecraft.item.ItemStack;
+import net.minecraft.tileentity.TileEntity;
+import net.minecraft.util.EnumChatFormatting;
+
+import forestry.api.apiculture.IBee;
+import forestry.api.apiculture.IBeeHousing;
+import forestry.api.apiculture.IBeeModifier;
+import gregtech.api.interfaces.metatileentity.IMetaTileEntity;
+import gregtech.api.interfaces.tileentity.IGregTechDeviceInformation;
+import gregtech.api.metatileentity.BaseMetaTileEntity;
+import gregtech.common.tileentities.machines.basic.GT_MetaTileEntity_IndustrialApiary;
+
+public class TileEntityApiamancersDrainerGT extends TileEntityApiamancersDrainerCommon
+        implements IGregTechDeviceInformation {
+
+    @Override
+    public boolean isGivingInformation() {
+        return true;
+    }
+
+    @Override
+    public String[] getInfoData() {
+        String aspects = essentia.aspects.entrySet().stream()
+                .map(
+                        e -> EnumChatFormatting.LIGHT_PURPLE + e.getKey().getName()
+                                + ": "
+                                + EnumChatFormatting.GREEN
+                                + e.getValue())
+                .collect(Collectors.joining(EnumChatFormatting.RESET + ", " + EnumChatFormatting.RESET));
+
+        return new String[] { //
+                EnumChatFormatting.DARK_PURPLE + "Apiamancer's Drainer" + EnumChatFormatting.RESET, //
+                aspect != null ? "Attuned: " + aspect.getName() : "Not attuned", //
+                "Stored Essentia:", //
+                aspects //
```

Comment:
Empty comment blocks can be removed

Reply:
They could but then the code becomes less readable in relation to what the scanner outputs to the player.
See below:
![image](https://github.com/GTNewHorizons/MagicBees/assets/7919610/968d2a5e-928c-4f2d-8e54-b8f942937204)